telemetry/definitions/commonDefinitions.json

```diff
@@ -1131,7 +1131,8 @@
                 "remoteDesktop",
                 "ssh",
                 "scp",
-                "ssm"
+                "ssm", 
+                "remoteWorkspace"
             ],
             "description": "Ways to connect to an EC2 Instance"
```

Comment:
Just for reference: this description is where we can explain certain fields, if we want to. E.g. in the future if someone wonders what the difference is between remoteDesktop vs remoteWorkspace, we could answer that here.